src/moveTo.js

```diff
@@ -121,7 +121,7 @@ const MoveTo = (() => {
    * @param  {object} options Custom options
    */
   MoveTo.prototype.move = function(target, options = {}) {
-    if (target !== 0 && !target) {
```

Comment:
Yes as you said `0` is falsy. If we do it the way you do `move` function will return without work
```js
// when target 0 
if (!0) {
  // will return 
}
```